cpu/stm32l1/periph/uart.c

```diff
@@ -119,15 +119,23 @@ static int init_base(uart_t uart, uint32_t baudrate)
     gpio_init(rx_pin, GPIO_IN);
     gpio_init_af(rx_pin, af);
 
+    /* cppcheck thinks dev may be NULL here, but the variable is
+     * assigned in all non-returning branches of the switch at the top of
+     * this function. */
```

Comment:
Maybe better add `assert(dev != NULL);` instead of adding five lines to suppress the warning?
In production `assert()` gets thrown away anyhow.